plugins/azure-devops/src/alpha/plugin.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  createApiExtension,
+  createApiFactory,
+  createPageExtension,
+  createPlugin,
+  discoveryApiRef,
+  identityApiRef,
+} from '@backstage/frontend-plugin-api';
+import { azureDevOpsApiRef, AzureDevOpsClient } from '../api';
+import {
+  compatWrapper,
+  convertLegacyRouteRef,
+} from '@backstage/core-compat-api';
+import {
+  createEntityCardExtension,
+  createEntityContentExtension,
+} from '@backstage/plugin-catalog-react/alpha';
+import { azurePullRequestDashboardRouteRef } from '../routes';
+
+/** @alpha */
+export const azureDevOpsApi = createApiExtension({
+  factory: createApiFactory({
+    api: azureDevOpsApiRef,
+    deps: { discoveryApi: discoveryApiRef, identityApi: identityApiRef },
+    factory: ({ discoveryApi, identityApi }) =>
+      new AzureDevOpsClient({ discoveryApi, identityApi }),
+  }),
+});
+
+/** @alpha */
+export const azureDevOpsPullRequestPage = createPageExtension({
+  defaultPath: '/azure-pull-requests',
+  routeRef: convertLegacyRouteRef(azurePullRequestDashboardRouteRef),
```

Comment:
Would you like to add a `TODO` here, so that you have a reminder that the legacy route converter may be deprecated in the future and this route will have to use the new routing system 🙂 ?